tdrs-backend/tdpservice/email/templates/system-admin-role-changed.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'base.html' %}
+{% block content %}
+<!-- Body copy -->
+<p style="color: #000000;">
+
+    <p>The following Admin User account for the TANF Data Portal (TDP) has been {{ action }}.</p>
```

Comment:
This reads a little bit weird with "added" and "removed" as the `action`. Could we do "assigned" and "unassigned"? Or change the sentence a little.

"The System Admin role has been (assigned/unassigned) for a user of the TANF Data Portal (TDP)"

just a suggested. sounds a little like the account has been deleted otherwise.